### DISCUSSION TITLE
Translation / Multi-Languge Support for PDF rendering

### DISCUSSION THREAD

**Discussion by Marcel2508** [ZUGFeRD/mustangproject#634] [Ideas]
Hey,
from what I can see visualisation as HTML supports multiple languages. PDF however not. What would be needed to support multiple languages for PDF as well?

**Comment by jstaerk** [comment]
Any contribution would be very much appreciated, in particular we need the french translation.
I believe the english translation is already prepared by the Kosit in library/src/main/resources/stylesheets  there is a de.xml and en.xml, we only need to find out how to activate it, can you help?
